code/game/objects/items/plushes.dm

```diff
@@ -772,6 +772,46 @@
 	lefthand_file = 'icons/mob/inhands/misc/plushes_lefthand.dmi' //todo: sort the god damn plushie inhands
 	righthand_file = 'icons/mob/inhands/misc/plushes_righthand.dmi'
 
+/obj/item/toy/plush/rilena
+	name = "RILENA:LMR Ri plushie"
```

Comment:
I've had some time to think about the phrasing of this a bit, and it might be better to leave out "RILENA:LMR" from the name, and just leave it for the desc. Same for all of these, hoodies too